application/src/main/java/org/togetherjava/tjbot/commands/tags/TagSystem.java

```diff
@@ -140,14 +139,16 @@ void putTag(String id, String content) {
      *
      * @return a set of all ids known to the system, not backed
```

Comment:
The documention is outdated with the code change you propose. It is not a set anymore and its also sorted, which is not mentioned here at all (hence should not be relied on!)